Played with the look of the Home Page

Oh-KPond opened the pull request:
I changed what the home page looked like and attempted at one point to add a photo. Although I looked at the JavaScript, I didn't change anything there. I did clean up some of the files though... and in that process lost functionality to the button. Ugh.  I can't see what I did. I'll look at it again tomorrow.

![image](https://cloud.githubusercontent.com/assets/11987546/8444506/446e8c6a-1f46-11e5-8068-1e543888f1b6.png)

Oh-KPond committed:
changed background color and attempted to put a background picture for newly created potluck. Lost the 'create new potluck' button. oops

Oh-KPond committed:
fount the 'create new potluck' button! Moved text around to try to make it more user friendly and appealing on the home page.

Oh-KPond committed:
styled home page and managed to lose button function

Evilnames commented:
I think you need to add more CATS to the page!

On Tue, Jun 30, 2015 at 4:37 PM Kate Pond notifications@github.com wrote:

> I changed what the home page looked like and attempted at one point to add
> a photo. Although I looked at the JavaScript, I didn't change anything
> there. I did clean up some of the files though... and in that process lost
> functionality to the button. Ugh. I can't see what I did. I'll look at it
> again tomorrow.
> 
> [image: image]
> 
> ## https://cloud.githubusercontent.com/assets/11987546/8444506/446e8c6a-1f46-11e5-8068-1e543888f1b6.png
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/Evilnames/Potluck/pull/6
> Commit Summary
> - changed background color and attempted to put a background picture
>   for newly created potluck. Lost the 'create new potluck' button. oops
> - fount the 'create new potluck' button! Moved text around to try to
>   make it more user friendly and appealing on the home page.
> - styled home page and managed to lose button function
> 
> File Changes
> - _M_ client/PotLuck.css
>   https://github.com/Evilnames/Potluck/pull/6/files#diff-0 (81)
> - _M_ client/PotLuck.html
>   https://github.com/Evilnames/Potluck/pull/6/files#diff-1 (7)
> - _M_ client/client.js
>   https://github.com/Evilnames/Potluck/pull/6/files#diff-2 (2)
> - _A_ client/images/potluck.jpg
>   https://github.com/Evilnames/Potluck/pull/6/files#diff-3 (0)
> - _M_ client/potlucklist.html
>   https://github.com/Evilnames/Potluck/pull/6/files#diff-4 (6)
> 
> Patch Links:
> - https://github.com/Evilnames/Potluck/pull/6.patch
> - https://github.com/Evilnames/Potluck/pull/6.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Evilnames/Potluck/pull/6.

Oh-KPond commented:
Yes, cats and potlucks really go well together. ;) Hahaha
On Jun 30, 2015 5:08 PM, "Alex" notifications@github.com wrote:

> I think you need to add more CATS to the page!
> 
> On Tue, Jun 30, 2015 at 4:37 PM Kate Pond notifications@github.com
> wrote:
> 
> > I changed what the home page looked like and attempted at one point to
> > add
> > a photo. Although I looked at the JavaScript, I didn't change anything
> > there. I did clean up some of the files though... and in that process
> > lost
> > functionality to the button. Ugh. I can't see what I did. I'll look at it
> > again tomorrow.
> > 
> > [image: image]
> > <
> > https://cloud.githubusercontent.com/assets/11987546/8444506/446e8c6a-1f46-11e5-8068-1e543888f1b6.png
> > 
> > ---
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/Evilnames/Potluck/pull/6
> > Commit Summary
> > - changed background color and attempted to put a background picture
> >   for newly created potluck. Lost the 'create new potluck' button. oops
> > - fount the 'create new potluck' button! Moved text around to try to
> >   make it more user friendly and appealing on the home page.
> > - styled home page and managed to lose button function
> > 
> > File Changes
> > - _M_ client/PotLuck.css
> >   https://github.com/Evilnames/Potluck/pull/6/files#diff-0 (81)
> > - _M_ client/PotLuck.html
> >   https://github.com/Evilnames/Potluck/pull/6/files#diff-1 (7)
> > - _M_ client/client.js
> >   https://github.com/Evilnames/Potluck/pull/6/files#diff-2 (2)
> > - _A_ client/images/potluck.jpg
> >   https://github.com/Evilnames/Potluck/pull/6/files#diff-3 (0)
> > - _M_ client/potlucklist.html
> >   https://github.com/Evilnames/Potluck/pull/6/files#diff-4 (6)
> > 
> > Patch Links:
> > - https://github.com/Evilnames/Potluck/pull/6.patch
> > - https://github.com/Evilnames/Potluck/pull/6.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/Evilnames/Potluck/pull/6.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Evilnames/Potluck/pull/6#issuecomment-117376106.

Evilnames commented on client/PotLuck.css:
```diff
@@ -21,41 +21,19 @@ label, input {
     position:absolute;
-    right:10px;
-    top:10px;
+    left:650px;
```
Why use left vs right in this instance?

Oh-KPond replied:
Because I could! :-D j/k   Thanks for pointing that out. I'm sure that I was just ignoring the code that was already there. I was distracted by the nonexistent cats.

... I see what you're saying. I suppose I'm unaware of the dimensions of the area I'm working with are, and on my screen it seems to make more sense to push it from the left side. Fascinating. I just looked at it from another browser.

[Hahaha.. I hope you're not getting a whole bunch of "updated comment" or anything. Oops if you are.]
